src/routes/canvas/toolbar/tools/Eraser.tsx

```diff
@@ -35,13 +36,15 @@ const pointEraserProps = {
 class Eraser extends Tool {
     static defaultProps = eraserProps;
     state = {
+        size: 5,
```

Comment:
After some tests, it seems that for the point eraser, a default value of 10 would work better.